crates/runtime/src/tracing_util.rs

```diff
@@ -50,9 +51,28 @@ pub fn dataset_registered_trace(
     info
 }
 
+// Format: View taxi_trips_vw registered, acceleration (duckdb)
+pub fn view_registered_trace(
+    table: &TableReference,
+    acceleration: Option<&acceleration::Acceleration>,
+) -> String {
+    let mut info = format!("View {table} registered");
+    if let Some(acceleration) = acceleration {
+        if acceleration.enabled {
+            info.push_str(&format!(
+                ", acceleration ({})",
+                dataset_acceleration_info(None, acceleration)
+            ));
+        }
+    }
+
+    info.push('.');
+    info
+}
+
 // Format: sqlite:file, 30s refresh, 1hr retention, fallback on source on empty result
 fn dataset_acceleration_info(
-    data_connector: &Arc<dyn DataConnector>,
+    data_connector: Option<&Arc<dyn DataConnector>>,
```

Comment:
Nit:

```suggestion
    data_connector: Option<&dyn DataConnector>,
```

It's a fairly strict type and should be able to be relaxed some. It's likely not an issue but it could help additional callers utilize this function.

Allows you to do use it like this:
```rust
use std::sync::Arc;

trait MyTrait {
    fn do_something(&self);
}

struct MyStruct;
impl MyTrait for MyStruct {
    fn do_something(&self) {
        println!("Doing something!");
    }
}

fn takes_option_trait_ref(opt_trait_ref: Option<&dyn MyTrait>) {
    if let Some(trait_ref) = opt_trait_ref {
        trait_ref.do_something();
    }
}

fn main() {
    let arc: Arc<dyn MyTrait> = Arc::new(MyStruct);
    takes_option_trait_ref(Some(&*arc));
}
```

Reply:
Thank you, updated `&Arc<dyn DataConnector>` -> `&dyn DataConnector`